sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -757,6 +757,32 @@ if (contextVersion > 1) {
         function(target) {
             return gl.getIndexedParameter(target, 0);
     });
+
+    debug("");
+    debug("Test getQueryParameter");
+    var buffer = gl.createBuffer();
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, buffer);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 64, gl.DYNAMIC_DRAW);
+    var query = gl.createQuery();
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buffer);
+    gl.beginQuery(gl.TRANSFORM_FEEDBACK_PRIMITIVES_WRITTEN, query);
+    gl.beginTransformFeedback(gl.TRIANGLES);
+    gl.drawArrays(gl.POINTS, 0, 3);
+    gl.endTransformFeedback();
+    gl.endQuery(gl.TRANSFORM_FEEDBACK_PRIMITIVES_WRITTEN);
+    shouldBe('gl.getQueryParameter(query, gl.QUERY_RESULT_AVAILABLE)', 'true');
```

Comment:
I have checked the latest spec. My patch has returned to the user agent's main loop. Is it right now? Please help review the code, thanks a lot. 
@kenrussell